homeassistant/components/voip/__init__.py

```diff
@@ -14,7 +14,7 @@
 from homeassistant.core import HomeAssistant
 from homeassistant.helpers import device_registry as dr
 
-from .const import DOMAIN
+from .const import CONF_SIP_PORT, DOMAIN
```

Comment:
You could just use `CONF_PORT` that we already have in `homeassistant.const`

Reply:
I considered that, but it's important that this is identified as the SIP port (as opposed to other ports we may let the user configure in the future).